default-parameters-es6/demo.js

```diff
@@ -0,0 +1,41 @@
+// Supply default values for parameters
+
+// Before, using ES5...
+function greetES5(message, to) {
+  // Test parameter values in the function body, specifying a default
+  to = to || 'DOM';
+  // OR if (typeof to === undefined) { to = 'DOM'; }
+  // OR if (arguments.to === 1) { to = DOM'; '}
+  ChromeSamples.log(message + ', ' + to);
+}
+greetES5('Good morning');
+greetES5('Sup', 'CSS');
+
+// After using ES2015/ES6...
+// Define default parameter values in the function head
+function greet(message, to = 'DOM') {
+  ChromeSamples.log(message + ', ' + to);
+}
+
+greet('Good morning');
+greet('Sup', 'CSS');
+
+// Another example
+function f(x, y = 12) {
+  // y is 12 if not passed (or passed as undefined)
+  return x + y;
+}
+
+f(3);
```

Comment:
Can you add in an additional example to hammer home how false-y values like `0` won't trigger the default? Something like

```js
// The default parameter won't be used since we're passing in a value, even though 0 is false-y.
f(5, 0);
```